src/test/unit/profile-tree.test.js

```diff
@@ -330,11 +330,13 @@ describe('unfiltered call tree', function() {
     describe('icons from the call tree', function() {
       it('upgrades http to https', function() {
         const { profile } = getProfileFromTextSamples(`
-          A:examplecom.js
+          A[lib:examplecom.js]
         `);
         const callTree = callTreeFromProfile(profile);
         const [thread] = profile.threads;
-        const hostStringIndex = thread.stringTable.indexForString('examplecom');
```

Comment:
The previous code was taking only the first word, so left out the `.js` part. The new code take it completely.